src/rules/selector-no-redundant-nesting-selector/README.md

```diff
@@ -59,3 +59,51 @@ p {
   }
 }
 ```
+
+## Options
+
+Some false alarms can be blocked by assigning keywords:
```

Comment:
This is not a good way of describing the option. 

A better way would be to say something like: if you are using Less or some other non-SCSS syntax, the warnings can be disabled by using this option.